packages/fxa-content-server/tests/functional/confirm.js

```diff
@@ -54,63 +45,21 @@ registerSuite('confirm', {
           .then(openPage(ENTER_EMAIL_URL, selectors.ENTER_EMAIL.HEADER))
           .then(fillOutEmailFirstSignUp(email, PASSWORD))
 
-          .then(testElementExists(selectors.CONFIRM_SIGNUP.HEADER))
+          .then(testElementExists(selectors.CONFIRM_SIGNUP_CODE.HEADER))
           .then(
             testElementTextInclude(
-              selectors.CONFIRM_SIGNUP.EMAIL_MESSAGE,
+              selectors.CONFIRM_SIGNUP_CODE.EMAIL_FIELD,
               email
             )
           )
-          .then(noSuchElement(selectors.CONFIRM_SIGNUP.LINK_OPEN_WEBMAIL))
 
-          .then(click(selectors.CONFIRM_SIGNUP.LINK_RESEND))
+          .then(click(selectors.CONFIRM_SIGNUP_CODE.LINK_RESEND))
 
           // the test below depends on the speed of the email resent XHR
           // we have to wait until the resent request completes and the
           // success notification is visible
           .then(testSuccessWasShown())
       );
     },
-
-    'sign up with a restmail address, get the open restmail button': function() {
```

Comment:
I guess technically the `open email button` is no longer used by moving to codes? Maybe a follow up to remove the feature?

Reply:
@ryanfeeley and I were discussing this last week, a new link to open the email provider may end up on the confirm yer code page.

Reply:
Opened https://github.com/mozilla/fxa/issues/3545